cli/roger_build.py

```diff
@@ -82,6 +82,12 @@ def main(self, settingObj, appObj, hooksObj, dockerUtilsObj, dockerObj, args):
             else:
                 repo = data.get('repo', args.app_name)
 
+            build_args = {}
+            if 'build-args' in data:
+                if 'environment' in data['build-args']:
```

Comment:
I don't think we need to tag an image with the environment. If a user doesn't use build-args, the image is free to run on any env. If the image use build-args, it could still be shared between some envs, not necessarily all.